.github/workflows/node-ci.yml

```diff
@@ -0,0 +1,151 @@
+name: node-ci
+
+on:
+  workflow_dispatch:
+  push:
+    tags-ignore:
+      - '**'
+    paths-ignore:
+      - "platform/android/**"
+      - "platform/ios/**"
+      - ".gitignore"
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+    paths-ignore:
+      - "platform/android/**"
+      - "platform/ios/**"
+      - ".gitignore"
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        os:
+          - macos-11.0
+          - macos-10.15
+          - ubuntu-20.04
+          - ubuntu-18.04
+    runs-on: ${{ matrix.os }}
+    env:
+      BUILDTYPE: Debug
```

Comment:
Should we build Release builds in the CI? (I have to check other pipelines what they do)

Reply:
Others are using `Debug` too. I think we should build debug builds for PRs, for `master` and tag builds build `Release`